acceptance-tests/features/steps/make_deed_effective_definitions.rb

```diff
@@ -0,0 +1,35 @@
+Given(/^I have made the created deed effective$/) do
+  sign_the_deed(@deed_id, 'SIGNATURE', '1')
+  make_deed_effective(@deed_id)
+end
+
+When(/^I get the deed from the api$/) do
+  @deed = get_deed_data(@deed_id)
+end
+
+Then(/^the signature is applied to the deed$/) do
+  assert(@deed['deed']['operative-deed']['registrars-signature'],
+         'Signature not applied')
+end
```

Comment:
Similar to below we are checking here that there is something in the registrars-signature in the deed object but not what it is exactly. Is it worth  expanding the assertion to check it is of value 'SIGNATURE'? Only thing I suppose is if this value of registrars signature changes then because we are hard coding the check this could make this test brittle.